solana/gatekeeper-lib/src/service/GatekeeperService.ts

```diff
@@ -29,6 +33,35 @@ export type GatekeeperConfig = {
   defaultExpirySeconds?: number;
 };
 
+/**
+ * Enables easier unit testing of gatekeeper-lib as we don't have to stub the imported function from gateway-ts,
+ * we can stub this function which is in the same lib.
+ */
+export const getGatewayTokenKeyForOwner = async (
```

Comment:
not sure I understand how stubbing a local function is very different from stubbing a lib function